interface/patient_file/summary/demographics_full.php

```diff
@@ -1158,9 +1158,7 @@ class='text'>(<?php echo xlt('Browse'); ?>)</a>
             var fval = $('#form_' + flds[i]).val();
             duplicateFieldsArray['#form_' + flds[i]] = fval;
         }
-        $(".sel2").select2({
-            <?php require($GLOBALS['srcdir'] . '/js/xl/select2.js.php'); ?>
-        });
```

Comment:
pretty sure this require is satisfied when select2 is brought in with the core header

Reply:
Select2 may have several instances each requiring its own library. That's why you see the way it is. So this is a no no!:)

Reply:
Also may have different setups.

Reply:
you already bring it in
https://github.com/openemr/openemr/blob/master/interface/patient_file/summary/demographics_full.php#L96

Reply:
I don't understand. You're removing the lines the permalink points to. What you're doing is removing the class link to select2 and the translations. Where is the class sel2 already inited?

Reply:
am removing lines 1161-1163, it's already required from line 96 right?

Reply:
I see your confusion. Each instance of select2 needs the translations pulled in by the required. Otherwise the select2 defaults untranslated would be used plus, all the html tags where the sel2 class is used prob won't work.

Reply:
ok, I'll revert and start another issue for it, try to figure out why that is breaking the returned entry from appearing in the select label

Reply:
Most likely the styling. I noticed this is the only select2 that is not using the bootstrap stylings. Checkout one of my select2's and emulate setup.
I should have done that when I was putting in the new ones. The one we are looking at was the only select2 when I put in the others. It has been there for awhile.

Reply:
do you have a screen shot of issue?

Reply:
Thinking about it @stephenwaite , do we need select2 for the two places it is used(sel2 class that is). It was put in for searching